doc/source/user/debugging.rst

```diff
@@ -0,0 +1,32 @@
+==============================
+Debugging
```

Comment:
Could you rename this to:
`Increase Ansible verbosity` or `Ansible tips`?

Reply:
Yeah I wanted a bit of feedback here. This may be just the debugging page and then a subsection on ansible or we could have different pages for debugging the different components.

Im ok with either, maybe separated is better in case some pages get too long